### DISCUSSION TITLE
Install failing

### DISCUSSION THREAD

**Discussion by gwww** [python-poetry/poetry#9347] [General]
Using the command: `curl -sSL https://install.python-poetry.org | python3 -` to install it fails with this:

```
dyld[77479]: Library not loaded: @executable_path/../lib/libpython3.12.dylib
  Referenced from: <C52F33D7-CDE0-3C12-B153-B6C7E6AA1600> /Users/glenn/.local/share/pypoetry/venv/bin/python3
  Reason: tried: '/Users/glenn/.local/share/pypoetry/venv/lib/libpython3.12.dylib' (no such file)

Traceback:

  File "<stdin>", line 923, in main
  File "<stdin>", line 562, in run
```
The `.local/share/pypoetry` directory did not exist before running the install. After is just contain a single directory `bin` which is empty.

I have python 3.12.1 installed through mise.

Is this a problem with my environment, the poetry installer, mise? I'm not too sure where to start looking.

**Comment by gwww** [comment]
I did find a solution on the mise project. Using `curl -sSL https://install.python-poetry.org | sed 's/symlinks=False/symlinks=True/' | python3 -` fixes the problem.

Is this a poetry installer problem?

**Reply by dijonkitchen** [reply]
This workaround works for me with the same problem. I'm also using python version management with `mise`. 

Not sure what the problem with symlinks is, but here's the only place in the code for it: https://github.com/python-poetry/install.python-poetry.org/blob/main/install-poetry.py#L317

**Reply by Buju77** [reply]
This workaround also worked for me. I am also using `mise` to manage my Python installation. Is this maybe a poetry installer problem not supporting symlinks?
